XRTK.WindowsMixedReality/Packages/com.xrtk.wmr/Controllers/WindowsMixedRealityController.cs

```diff
@@ -243,10 +243,10 @@ private void UpdateGripData(InteractionSourceState interactionSourceState, Mixed
                         interactionSourceState.sourcePose.TryGetPosition(out currentGripPosition, InteractionSourceNode.Grip);
                         interactionSourceState.sourcePose.TryGetRotation(out currentGripRotation, InteractionSourceNode.Grip);
 
-                        if (MixedRealityToolkit.Instance.MixedRealityPlayspace != null)
+                        if (MixedRealityToolkit.CameraSystem.CameraRig.PlayspaceTransform != null)
```

Comment:
Was this the only controller we missed in the update?

Reply:
That I'm aware of, although I didn't validate the Lumin platform (but figured you had :D )

Reply:
Interesting that we're doing the spatial transformation for this controller but not the others.

Reply:
I think it is simply leftover from the original implementation, or something possibly to do with the WMR implementation requiring it maybe?

Reply:
@SimonDarksideJ should these be pointing to the `BodyTransform`?

Reply:
I'll test and update. in architecture terms, probably.  Ultimately though, everything is at 0, so it wouldn't matter.